.mkosi/mkosi.opensuse

```diff
@@ -0,0 +1,62 @@
+# SPDX-License-Identifier: LGPL-2.1+
+#
+# Copyright © 2018 Michal Koutný
```

Comment:
We removed (almost all) copyright lines, see #9320. I'd prefer to not add this one.

Reply:
No problem, removed in a force push.